hugr-core/src/export.rs

```diff
@@ -0,0 +1,726 @@
+//! Exporting HUGR graphs to their `hugr-model` representation.
```

Comment:
Given the differences between the core rust representation and the hugr model, it would be nice to list what's different here. (node regions, unsupported optypes, ...)

Reply:
I'm in the process of writing these things up as comments to the issue https://github.com/CQCL/hugr/issues/1433

Reply:
I've added the ones that I am aware of.